inc/form.class.php

```diff
@@ -643,32 +643,37 @@ public function showWizard() : void {
    public function showFormList(int $rootCategory = 0, string $keywords = '', bool $helpdeskHome = false) : array {
       global $DB, $TRANSLATE;
 
-      $table_cat          = getTableForItemType('PluginFormcreatorCategory');
-      $table_form         = getTableForItemType('PluginFormcreatorForm');
-      $table_fp           = getTableForItemType('PluginFormcreatorForm_Profile');
-      $table_section      = getTableForItemType('PluginFormcreatorSections');
-      $table_question     = getTableForItemType('PluginFormcreatorQuestions');
-      $table_formLanguage = getTableForItemType(PluginFormcreatorForm_Language::class);
+      $table_cat      = getTableForItemType(PluginFormcreatorCategory::class);
+      $table_form     = getTableForItemType(PluginFormcreatorForm::class);
+      $table_fp       = getTableForItemType(PluginFormcreatorForm_Profile::class);
+      $table_section  = getTableForItemType(PluginFormcreatorSections::class);
+      $table_question = getTableForItemType(PluginFormcreatorQuestions::class);
 
       $categoryFk = PluginFormcreatorCategory::getForeignKeyField();
 
       $order         = "$table_form.name ASC";
 
-      $dbUtils = new DbUtils();
-      $entityRestrict = $dbUtils->getEntitiesRestrictCriteria($table_form, "", "", true, false);
-      if (count($entityRestrict)) {
-         $entityRestrict = [$entityRestrict];
-      }
-      $where_form = [
-         'AND' => [
-            "$table_form.is_active" => '1',
-            "$table_form.is_visible" => '1',
-            "$table_form.is_deleted" => '0',
-            'OR' => [
-               "$table_form.language" => [$_SESSION['glpilanguage'], '0', '', null],
-               "$table_formLanguage.name" => $_SESSION['glpilanguage']
-            ],
-         ] + $entityRestrict
```

Comment:
the above WHERE clause must always be used when searching for forms useable by a requester

restriction 
- by entity
- active form
- not deleted form
- visible form
- language matching the curent user's language (form translation included)